app/decorators/swagger/route_decorator.rb

```diff
@@ -5,6 +5,7 @@ def to_swagger(api_gateway_integration)
     {
       tags: [resource.name],
       description: description,
+      operationId: swagger_operation_id,
```

Comment:
I did not find out how to check for uniqueness in a project. I found for the resource and for the database, but not the project. I do not master scopes enough to do so.

Reply:
```
class Route
  validate :operation_id_unique_in_project
  
  def operation_id_unique_in_project
      return if operation_id.blank?
      errors.add(:operation_id, :must_be_unique_in_project) if project.routes.where.not(id: id).where(operation_id: operation_id).any?
  end
end
```

Reply:
It worked, thanks.